Week-4/W4Exercise4.3/index.js

```diff
@@ -0,0 +1,22 @@
+const Fib = {
+  n: 7,
```

Comment:
instead of hard coding this, should create Fib as a function and call that with an arg.